frontend/src/Footer/Footer.js

```diff
@@ -17,20 +17,20 @@ const Footer = () => {
             <FooterLink href="/home">Home</FooterLink>
           </Column>
           <Column>
-            <FooterLink href="/blogs">Blogs</FooterLink>
+            <FooterLink href="/contact">ContactUs</FooterLink>
           </Column>
           <Column>
-            <FooterLink href="/signin">Signin</FooterLink>
+            <FooterLink href="/signup">Register</FooterLink>
           </Column>
           <Column>
-            <FooterLink href="/signup">Signup</FooterLink>
+            <FooterLink href="/login">Login</FooterLink>
           </Column>
         </Row>
-        <h4
-          style={{ color: "#F5EFE7", textAlign: "center", marginTop: "20px" }}
```

Comment:
Can we put all the css style code to a new CSS file for "Footer". For example, create a "Footer.css" and bundle Footer.css with Footer.js.